Dockerfile.base

```diff
@@ -81,7 +68,7 @@ ENV PLAYWRIGHT_SKIP_DOWNLOAD=false
 ENV DISPLAY=:99
 
 RUN apt update \
-	&& apt-get install -y \
+	&& apt install -y --no-install-recommends \
```

Comment:
This is often risky, but it has paid off in our case as the only difference between regular `apt` and one with `--no-install-recommends` is `xauth`.